src/Symfony/Component/Security/Core/Tests/Authorization/Voter/AbstractVoterTest.php

```diff
@@ -65,16 +65,6 @@ public function testVoteLegacy(array $attributes, $expectedVote, $object, $messa
 
         $this->assertEquals($expectedVote, $voter->vote($this->token, $object, $attributes), $message);
     }
-
-    /**
-     * @group legacy
-     * @expectedException \BadMethodCallException
-     */
-    public function testNoOverriddenMethodsThrowsException()
-    {
-        $voter = new AbstractVoterTest_NothingImplementedVoter();
-        $voter->vote($this->token, new \stdClass(), array('EDIT'));
-    }
```

Comment:
Well, then the code is currently broken as this test would not pass. Can you point me into the direction of the code that would be responsible for this? Then I can readd the test and fix the code.